PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/RunProfileStateWithAppInsightsEvent.kt

```diff
@@ -50,4 +52,9 @@ abstract class RunProfileStateWithAppInsightsEvent(val uuid: String,
         val postEventProps = executor?.let { getPostEventProperties(it, addedEventProps) }
         EventUtil.logErrorWithComplete(operation, errorType, exp, postEventProps, null)
     }
+
+    fun createInfoEventWithComplete(executor: Executor?, addedEventProps: Map<String, String>?) {
+        val postEventProps = executor?.let { getPostEventProperties(it, addedEventProps) }
```

Comment:
Use empty maps instead of `null`

```suggestion
        val postEventProps = executor?.let { getPostEventProperties(it, addedEventProps) } ?: emptyMap()
```